common-lib/src/cmr/common/generics.clj

```diff
@@ -6,4 +6,5 @@
 (defn approved-generic?
   "Check to see if a requested generic is on the approved list"
   [schema version]
-  (some #(= version %) (schema (cfg/approved-pipeline-documents))))
+  (when (and schema version)
```

Comment:
I feel like we would have known this had we tested it in a file. Should we write a test?

Reply:
No, the ingest system-int-test now takes care of it.  A null pointer exception was being thrown when I was testing it.